js/mobile_menu.js

```diff
@@ -13,4 +13,231 @@ function menuPop() {
+    <ul>
+      <li>
+        <a href="#">${presentProject.techStack[0]}</a>
+      </li>
+      <li>
+        <a href="#">${presentProject.techStack[1]}</a>
+      </li>
+      <li>
+        <a href="#">${presentProject.techStack[2]}</a>
+      </li>
+      <li>
+        <a href="#">${presentProject.techStack[3]}</a>
+      </li>
+    </ul>
+    <button id=${presentProject.Name} onclick="buttonClick(this.id)" type="button">See project</button>
+  </div>
+</section>`;
+
+  return worksHeadline;
+}
+
+const showSections = () => {
+  const divCard = worksContent.map((item) => {
+    if (item.id === 1) {
+      return `<section class="center-content">
+    <section class="content-desc">
+      <img src=${item.img} alt="" />
+      <div class="top-left1">
+      <h2>${item.worksName}</h2>
+      <p>
+       ${item.worksContent}
+      </p>
+      <ul>
+        <li>
+          <a href="#">${item.techStack[0]}</a>
+        </li>
+        <li>
+          <a href="#">${item.techStack[1]}</a>
+        </li>
+        <li>
+          <a href="#">${item.techStack[2]}</a>
+        </li>
+        </ul>
+        <button id=${item.id} onclick="buttonClick(this.id)" type="button">See project</button>
+      </div>
+    </section>
+  </section>`;
+    }
+    return `<section class="center-content">
+  <section class="content-desc">
+    <img src=${item.img} alt="" />
+    <div class="top-left">
+      <h2>${item.worksName}</h2>
+      <p>
+       ${item.worksContent}
+      </p>
+      <ul>
+        <li>
+          <a href="#">${item.techStack[0]}</a>
+        </li>
+        <li>
+          <a href="#">${item.techStack[1]}</a>
+        </li>
+        <li>
+          <a href="#">${item.techStack[2]}</a>
+        </li>
+      </ul>
+      <button id=${item.id} onclick="buttonClick(this.id)" type="button">See project</button>
+    </div>
+  </section>
+</section>`;
+  });
+
+  return divCard;
+};
+
+function popupWin() {
+  const pop = `<div class="popup">
+  <div class="popup_content">
+    <div class="titleContent">
+      <div class="heading">
+        <h2 id="headingSection"></h2>
+      </div>
+      <img id="closeButton" onclick="closed()" src="images/icons/close.svg" alt="close">
+    </div>
+    <ul>
+      <li>
+        <a href="#">CSS</a>
+      </li>
+      <li>
+        <a href="#">HTML</a>
+      </li>
+      <li>
+        <a href="#">BOOTSTRAP</a>
+      </li>
+      <li>
+        <a href="#">RUBY</a>
+      </li>
+    </ul>
+    <div class="popWindow">
+      <div id="popup_info">
+        <img id="popup-img" src="images/icons/Snapshoot_Portfolio.jpg" alt="Snapshop_Portfolio">
+      </div>
+      <div class ="all_info">
+        <p id="projectDesc"></p>
+        <div class="bottom_section">
+          <button id="projectSeven" type="button">See project <img src="images/icons/see_live.png" alt="" class="btnIcon"></button>
+          <button id="projectSeven" type="button">See project <img src="images/icons/git.svg" class="btnIcon"> </button>
```

Comment:
- [ ] I'd suggest you replace the `button` elements here with an anchor tag so that you can easily place the address for each project also please make sure one of these buttons is a link to the live version of the project because both of them are currently "See project" buttons. ✔